src/models/methods/subscribe.ts

```diff
@@ -115,10 +115,25 @@ export interface ConsensusStream extends BaseStream {
   consensus: "open" | "establish" | "accepted";
 }
 
+export interface PathFindStream {
+  type: "path_find";
+  source_account: string;
+  destination_account: string;
+  destination_amount: Amount;
+  full_reply: boolean;
+  id: any;
+  send_max?: Amount;
+  alternatives: {
+    paths_computed: Path[];
+    source_amount: string;
```

Comment:
I think the source_amount should be an "Amount" based on this documentation for PathFindResponse (Which might not be the same thing you're going off of here, so I could be wrong!) - https://xrpl.org/path_find.html#path_find